config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "https://guides.xivresources.com/"
+baseURL = "https://thebalanceffxiv.com/"
 languageCode = "en-us"
 title = "Balance Guides"
```

Comment:
Do we need to change this title? probably for later when we add the <title> tag in the head.html

Reply:
That's definitely a later thing.